install/kubernetes/helm/istio/charts/pilot/templates/gateway.yaml

```diff
@@ -74,3 +74,72 @@ spec:
     hosts:
     - "*"
 {{- end }}
+---
+{{- if .Values.global.multiClusterUsingGateway }}
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istio-multicluster-gateway
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    istio: mcgateway
```

Comment:
Can you make this customizable ? Also the ports. 

I think in some cases users may want to run the 'cross cluster' gateway on the same load balancer and IP as the normal ingress gateway. Both IPs and load balancers can be expensive.
We probably want to use a different port to avoid confusion and make it easy to firewall/etc.

Reply:
We cannot use the same port as ingress gateway because I am overriding the filter chain with a special filter. May be a different port, sure.